packages/admin/src/cli.ts

```diff
@@ -56,16 +77,29 @@ const COMMON_COMMAND_ARGUMENTS = {
     demandOption: true,
     describe: 'Withdrawal request ID',
   },
+  expirationTimestamp: {
+    type: 'number',
+    demandOption: true,
+    describe: 'Timestamp at which the whitelisting of the user will expire',
+  },
+  whitelistStatusPastExpiration: {
+    type: 'boolean',
+    demandOption: true,
+    describe: 'Whitelist status that the user will have past expiration',
```

Comment:
Here's a link to a conversation where Emanuel asked about this too: https://api3workspace.slack.com/archives/C027Y2FAV0S/p1630408806006200

We basically need to keep the whitelisting set to true even after the expiration time elapses because the idea is to always try to give access to requesters rather than blocking access. I think the use case is more obvious with the token lock based authorizer where the sponsor locks tokens and the requester should be able to make requests even after expiration date as long as tokens remains locked.